ui/tests/integration/components/http-requests-bar-chart-small-test.js

```diff
@@ -0,0 +1,25 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+const FILTERED_HTTPS_REQUESTS = [
+  { start_time: '2018-11-01T00:00:00Z', total: 5500 },
+  { start_time: '2018-12-01T00:00:00Z', total: 4500 },
+  { start_time: '2019-01-01T00:00:00Z', total: 5000 },
+  { start_time: '2019-02-01T00:00:00Z', total: 5000 },
+];
+
+module('Integration | Component | http-requests-bar-chart-small', function(hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.set('filteredHttpsRequests', FILTERED_HTTPS_REQUESTS);
+  });
+
+  test('it renders', async function(assert) {
+    await render(hbs`<HttpRequestsBarChartSmall @counters={{filteredHttpsRequests}}/>`);
+
+    assert.dom('.http-requests-bar-chart-small').exists();
```

Comment:
similar to your test about only displaying 12 months of data, i think it'd be good to assert that there are at least 4 bars here